app/views/layouts/shared/_navbar.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class='navbar'>
+    <h2>Recipe Application</h2>
+    <div class='sign'>
+        <% if user_signed_in? %>
+        <div class='menu'>
+        <ul>
+            <li><%= link_to "Home", root_path %></li>
+            <li><%= link_to "Recipes", recipes_path %></li>
+            <li><%= link_to "Users", users_path %></li>
+            <li><%= link_to "Public recipes", public_recipes_path %></li>
+            
+        </ul>
+    </div>
```

Comment:
- [ ] You did a good job working on this milestone! 💯 However, I noticed that you don't have a link for the user's `Foods` page. According to the requirements:

![image](https://github.com/gsmalik030/recipe_app/assets/77069011/cd20525f-0fa7-479c-b3d2-6856e1ccbc10)

Please make sure to include it! 🤓 🔧